nfs-ganesha/build/build_deb

```diff
@@ -0,0 +1,90 @@
+#! /usr/bin/bash
+set -ex
+
+# Only do actual work when we are a DEB distro
+if test "$DISTRO" != "debian" -a "$DISTRO" != "ubuntu"; then
+    exit 0
+fi
+
+
+## Install any setup-time deps
+# TODO -- upload install-deps.sh script that installs the setup time deps
+# to upstream repo or update this section to install these packages
+
+# We need this for mk-build-deps
+sudo apt-get install -y equivs cmake git
+
+## Setup the pbuilder
+# TODO remove if you do not want to use pbuilders
+setup_pbuilder
+
+
+## Get some basic information about the system and the repository
+# TODO -- update this to get the proper VERSION/REVISION
+DEB_ARCH=$(dpkg-architecture -qDEB_BUILD_ARCH)
+# ALI TO DO GET VERSION
+VERSION="$(./get-version.sh)"
+
+
+## Build the source tarball
+# ALI TO DO: find out if this can be set in config/definitions/nfs-ganesha.yaml
+NFS_GANESHA_BRANCH="next"
+
+git checkout $NFS_GANESHA_BRANCH
+git submodule update --init || git submodule sync
+
+mkdir build
+cd build
+
+# make source tarball
+cmake -DCMAKE_BUILDER_TYPE=Maintainer ../src && make dist || exit 0
+
+
+## Prepare the debian files
+# TODO -- Make sure the debian folder is tracked upstream
+# Bump the changelog
+dch -v "$VERSION" "New release ($VERSION)"
+
+# Create .dsc and source tarball
+sudo dpkg-buildpackage -S -us -uc
+
+
+## Build with pbuilder
+echo "Building debs"
+
+PBUILDDIR="/srv/debian-base"
+
+sudo pbuilder --clean
+
+mkdir -p dist/deb
+
+# ALI TO DO: find out if there's a env var for CEPH_BRANCH
```

Comment:
there isn't one available unless you make it a parameter or define it as a variable for this job